private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go

```diff
@@ -73,7 +73,7 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		&f.Username,
 		usernameFlagName,
 		"",
-		"The username to use. This command prompts for a username by default",
+		"The username to use. If provided, the command will validate that the token is associated with this username",
```

Comment:
I would hide and deprecate this flag.

Reply:
Hidden and reverted text to just "The user to use".